src/engine/enginedeck.cpp

```diff
@@ -34,6 +34,7 @@ EngineDeck::EngineDeck(const ChannelHandleAndGroup& handle_group,
         : EngineChannel(handle_group, defaultOrientation),
           m_pConfig(pConfig),
           m_pEngineEffectsManager(pEffectsManager ? pEffectsManager->getEngineEffectsManager() : NULL),
+          m_pInputConfigured(new ControlObject(ConfigKey(getGroup(), "input_enabled"))),
```

Comment:
The previous code that I'm refactoring away needed a way to check if an input was configured for the deck -- microphone/aux have an "enabled" control that indicates this so I named it input_enabled for uniformity.